acceptance/test.toml

```diff
@@ -0,0 +1,2 @@
+Local = true
```

Comment:
Can you add a comment here that this is the global configuration for all tests and that tests are only run locally by default and not as integration tests?

Reply:
added a comment